### DISCUSSION TITLE
Actividad Docker - GRUPO 3

### DISCUSSION THREAD

**Discussion by BogadoLucasDemian** [frlp-utn-ingsoft/catedra#112] [Show and tell]
https://github.com/BogadoLucasDemian/vetsoft

**Comment by rodriguezemautn** [comment]
Revisen el repo. No esta la actividad

**Reply by BogadoLucasDemian** [reply]
Perdón, confundí el repo porque el otro se llama Ing. de Software, ya lo actualice
